diff --git a/index-browser.js b/index-browser.js
--- a/index-browser.js
+++ b/index-browser.js
@@ -1,3 +1,3 @@
-const prisma = require('.prisma/client/index-browser')
+const prisma = require('./.prisma/client/index-browser')
 
 module.exports = prisma
diff --git a/index.js b/index.js
--- a/index.js
+++ b/index.js
@@ -1,4 +1,4 @@
 module.exports = {
   // https://github.com/prisma/prisma/pull/12907
-  ...require('.prisma/client/default'),
+  ...require('./.prisma/client/default'),
 }
